CHANGELOG.md

```diff
@@ -124,6 +124,8 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 * `jj describe` can now update the description of multiple commits.
 
+* When reconfiguring the author, warn that the working copy won't be updated
```

Comment:
Nit: This should be shifted up to the unreleased section since the last release.

Reply:
Great catch, thanks!